client/components/woopay/save-user/style.scss

```diff
@@ -88,6 +90,10 @@
 				display: flex;
 			}
 
+			span {
+				color: #333 !important;
```

Comment:
One thing to call out here is that the original issue is just that we only defined a background color on the flyout and not a text color on it. This led to the theme's text color being applied (white), but the background used our styles (also white). All that to say I'm not sure we need an important here or if we do, we should also have it on the background color as well to ensure contrast. Let me know your thoughts on getting those two things in sync?

Reply:
>All that to say I'm not sure we need an important here

Good callout! I'm happy removing important since it will now require intention to change the color.

Reply:
Perfect, thanks for addressing that!

LGTM!